packages/algolia-experiences/src/get-configuration.ts

```diff
@@ -0,0 +1,120 @@
+import type { Settings } from './get-information';
+import type { Block, Configuration } from './types';
+
+export function fetchConfiguration(
+  ids: string[],
+  settings: Settings
+): Promise<Configuration[]> {
+  return Promise.all(
+    ids.map((id) =>
+      getExperience({
+        id,
+        ...settings,
+      })
+    )
+  );
+}
+
+export type Experience = {
+  id: string;
+  name: string;
+  indexName: string;
+  blocks: Block[];
+  createdAt: string;
+  updatedAt: string;
+};
+
+const LOCAL = false;
```

Comment:
Do you intend on modifying this value during rollup for the development bundle?

Reply:
maybe, for now this allows you to easily toggle between the two without commented out code.